tyr/servers/server.py

```diff
@@ -136,8 +158,9 @@ def configure(self):
         self.establish_route53_connection()
 
         if self.ami is None:
-            self.log.warn('No AMI provided')
-            self.ami = 'ami-8fcee4e5'
+            self.log.warn('No AMI provided, searching for latest one...')
+            self.ami = self.get_latest_ami(self.ami)
```

Comment:
Good points. It was put in as AWS kill of AMIs, you can bypass this code by hardcoding the AMI as a parameter. Ill leave it to the group to decide if this is still wanted or not (personally I agree that this should probably be removed).